src/ethr-did-resolver.js

```diff
@@ -216,9 +237,18 @@ function getResolver (conf = {}) {
     }
     return { owner, history }
   }
-  async function resolve (did, parsed) {
-    if (!parsed.id.match(/^0x[0-9a-fA-F]{40}$/)) throw new Error(`Not a valid ethr DID: ${did}`)
-    const { owner, history } = await changeLog(parsed.id)
+  async function resolve(did, parsed) {
+    const fullId=parsed.id.match(/^(.*)?(0x[0-9a-fA-F]{40})$/)
+    if (!fullId)
+      throw new Error(`Not a valid ethr DID: ${did}`)
+
+    const id=fullId[2];
+    const network = (!fullId[1]) ? 'mainnet' : fullId[1].slice(0,-1);
+
+    if(!networks[network])      
+      throw new Error(`No conf for network: ${network}`)
+
+    const { owner, history } = await changeLog(id)
     return wrapDidDocument(did, owner, history)
```

Comment:
I believe the DID document for `did:ethr:0xaddress` and `did:ethr:mainnet:0xaddress` should be identical.
If this is not the case then the kotlin implementation should also be adapted to fit